### PR TITLE
Change device sessions

### PR DESCRIPTION
**Pull request by abhint**
- Change device sessions
   -  Device name - https://github.com/LonamiWebs/Telethon/commit/06a9528dc060f38c4f6ff7151bf5c74acb0c45e1
   - System version  - https://github.com/LonamiWebs/Telethon/commit/71cc3dffa662c1533b771440d890a3e18e233f08

### CODE REVIEW

File: telethon/client/telegrambaseclient.py [hunk 1]
```diff
@@ -358,12 +357,14 @@ def __missing__(self, key):
         system = platform.uname()
 
         if system.machine in ('x86_64', 'AMD64'):
-            default_device_model = 'PC 64bit'
-        elif system.machine in ('i386','i686','x86'):
-            default_device_model = 'PC 32bit'
+            default_device_model = 'Telethon 64bit'
+        elif system.machine in ('i386', 'i686', 'x86'):
+            default_device_model = 'Telethon 32bit'
```
Comment (Lonami) [inline telethon/client/telegrambaseclient.py:362]: What's the rationale behind this change?

File: telethon/client/telegrambaseclient.py [hunk 2]
```diff
@@ -358,12 +357,14 @@ def __missing__(self, key):
...
         else:
             default_device_model = system.machine
-        default_system_version = re.sub(r'-.+','',system.release)
+
+        # default_system_version = re.sub(r'-.+', '', system.release)
+        default_system_version = system.version
```
Comment (Lonami) [inline telethon/client/telegrambaseclient.py:367]: My version is:

> #1 SMP Thu Jan 20 15:38:18 UTC 2022

Is this something Telegram actually accepts?

**Comment by Lonami** [comment]
Hi @abhint, any feedback regarding my comments?

**Comment by abhint** [comment]
@Lonami Sorry for the late reply 🧑‍🎓

**Comment by Lonami** [comment]
I'm going to close this for the time being. We can revive it in a future.
